tests/test_fixture.py

```diff
@@ -1,40 +1,185 @@
+            }
+        },
+        {
+            "assists": {
+                "a": [{"value": 1, "element": 221}],
+                "h": [{"value": 1, "element": 295}, {"value": 1, "element": 297}],
+            }
+        },
+        {"own_goals": {"a": [], "h": []}},
+        {"penalties_saved": {"a": [], "h": []}},
+        {"penalties_missed": {"a": [], "h": []}},
+        {
+            "yellow_cards": {
+                "a": [{"value": 1, "element": 226}],
+                "h": [{"value": 1, "element": 304}, {"value": 1, "element": 481}],
+            }
+        },
+        {"red_cards": {"a": [], "h": []}},
+        {
+            "saves": {
+                "a": [{"value": 4, "element": 213}],
+                "h": [{"value": 3, "element": 282}],
+            }
+        },
+        {
+            "bonus": {
+                "a": [{"value": 1, "element": 234}],
+                "h": [{"value": 3, "element": 286}, {"value": 2, "element": 302}],
+            }
+        },
+        {
+            "bps": {
+                "a": [
+                    {"value": 25, "element": 234},
+                    {"value": 23, "element": 221},
+                    {"value": 16, "element": 213},
+                    {"value": 16, "element": 215},
+                    {"value": 15, "element": 225},
+                    {"value": 14, "element": 220},
+                    {"value": 13, "element": 227},
+                    {"value": 13, "element": 231},
+                    {"value": 12, "element": 219},
+                    {"value": 10, "element": 233},
+                    {"value": 6, "element": 226},
+                    {"value": 5, "element": 228},
+                    {"value": 3, "element": 492},
+                    {"value": 2, "element": 236},
+                ],
+                "h": [
+                    {"value": 30, "element": 286},
+                    {"value": 29, "element": 302},
+                    {"value": 24, "element": 297},
+                    {"value": 22, "element": 295},
+                    {"value": 16, "element": 289},
+                    {"value": 15, "element": 282},
+                    {"value": 15, "element": 292},
+                    {"value": 13, "element": 291},
+                    {"value": 13, "element": 305},
+                    {"value": 13, "element": 481},
+                    {"value": 8, "element": 304},
+                    {"value": 4, "element": 298},
+                    {"value": 3, "element": 303},
+                    {"value": -2, "element": 306},
+                ],
+            }
+        },
+    ],
+    "team_h_difficulty": 3,
+    "team_a_difficulty": 4,
+    "code": 987597,
+    "kickoff_time": "2018-08-10T19:00:00Z",
+    "team_h_score": 2,
+    "team_a_score": 1,
+    "finished": True,
+    "minutes": 90,
+    "provisional_start_time": False,
+    "finished_provisional": True,
+    "event": 1,
+    "team_a": 11,
+    "team_h": 14,
+}
+
+
+@pytest.fixture()
```

Comment:
Can't you use the fixture in `conftest.py`?

Reply:
conftest.py is intended for shared fixtures. However, I found that they are independent and thus it made more sense to me to move it to respective class test file. Moreover, when testing the initialization it reduces redundancy of test data as it does not need to be repeated nor imported.

I can change it but I think it is not necessary to put all fixtures to common file if they are not shared across different tests.